iTunesUDataPurge/config.properties.template

```diff
@@ -0,0 +1,3 @@
+iTunesUSiteId=iTunesU_site_id
+sharedSecret=iTunesU_site_sharedSecret
+admin_credential=admin_credential
```

Comment:
With the new properties file, do the Securities Files on Deluxe need to be updated?